io.github.jpochyla.Psst.yaml

```diff
@@ -0,0 +1,38 @@
+app-id: io.github.jpochyla.Psst
+runtime: org.gnome.Platform
+runtime-version: '44'
+sdk: org.gnome.Sdk
+sdk-extensions:
+- org.freedesktop.Sdk.Extension.rust-stable
+command: psst-gui
+finish-args:
+- "--share=ipc"
+- "--share=network"
+- "--socket=fallback-x11"
+- "--socket=wayland"
+- "--device=dri"
+- "--socket=pulseaudio"
+
+modules:
+
+- name: "psst-gui"
+  buildsystem: simple
+  build-options:
+    append-path: "/usr/lib/sdk/rust-stable/bin"
+    env:
+      CARGO_HOME: "/run/build/psst-gui/cargo"
```

Comment:
It might be possible that setting this env var is not necessary.

Reply:
it likely is. it's not a vendored tarball.